src/core/cgroup.c

```diff
@@ -952,6 +952,15 @@ static void cgroup_apply_unified_cpu_weight(Unit *u, uint64_t weight) {
         (void) set_attribute_and_warn(u, "cpu", "cpu.weight", buf);
 }
 
+static void cgroup_apply_unified_cpu_idle(Unit *u, uint64_t weight) {
+        int r;
+
+        r = cg_set_attribute("cpu", u->cgroup_path, "cpu.idle", one_zero(weight));
+        if (r < 0 && (r != -ENOENT || weight))
+                log_unit_full_errno(u, LOG_LEVEL_CGROUP_WRITE(r), r, "Failed to set '%s' attribute on '%s' to '%" PRIu64 "': %m",
+                                    "cpu.idle", empty_to_root(u->cgroup_path), weight);
```

Comment:
log message is confusing, it should probably show the result of `one_zero(weight == CGROUP_WEIGHT_IDLE)` and not the literal value of `weight`